Src/BootCamp.Chapter/Person.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace BootCamp.Chapter
+{
+    internal class Person
+    {
+        private string _name;
+        private Account _balance;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_balance' or refactor the code to use its value.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)